app/controllers/staff/users_controller.rb

```diff
@@ -1,24 +1,24 @@
 class Staff::UsersController < Staff::BaseController
   def index
+    authorize :user, :index?
```

Comment:
Only fund_managers and administrators should be able to see this page so we call `authorize` first before spending time querying.